ambariclient/base.py

```diff
@@ -396,7 +397,11 @@ class Model(object):
     min_version = OLDEST_SUPPORTED_VERSION
 
     def __init__(self, parent, data=None):
-        self._data = data if data is not None else {}
+        if data is None:
+            data = {}
+
+        self._data = dict((key, value) for key, value in six.iteritems(data)
```

Comment:
I'm concerned that this might expose bugs in currently ill-defined models that are missing some fields, which is entirely possible.  What problem were you solving with this change?